include/animation/linear_animation_instance.hpp

```diff
@@ -12,18 +12,36 @@ namespace rive
 		LinearAnimation* m_Animation = nullptr;
 		float m_Time;
 		int m_Direction;
+		bool m_didLoop;
```

Comment:
Total nitpick, but we m_CamelCase the other members, maybe do the same for this one.